vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -98,18 +84,12 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
             "mcr.microsoft.com/oss/kubernetes/pause:3.9",
             # CSI. Owner: andyzhangx (Andy Zhang)
-            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.5.0",
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.6.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.5.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.2",
```

Comment:
keep this version

Reply:
Reverted a13a3d6a2 - chore: Delete old CSI images